examples/gateway/00-crds.yaml

```diff
@@ -5845,6 +5845,48 @@ spec:
                         type: object
                       maxItems: 8
                       type: array
+                    timeouts:
```

Comment:
I'm a little surprised this is included in v0.8.1 -- looks like the tag was created on the `main` branch rather than `release-0.8`, not sure if that was intentional or not.

Reply:
Yeah I agree the changelog and the release branch do not include these commits, something to bring up to the maintainers

Reply:
Sounds like these are going to be left in, since this is the experimental channel and the fields will be included in 1.0 anyway, so I think we're OK to go ahead and merge here.